pom.xml

```diff
@@ -128,12 +128,33 @@
   <build>
     <plugins>
       <plugin>
+        <groupId>org.codehaus.mojo</groupId>
```

Comment:
Yes. This is more a safeguard against developers who use Jdk 1.8 to develop and compile, which can't detect usage of unavailable APIs in minor versions.